### ISSUE TITLE
Search does not seem to search titles.

### ISSUE THREAD

**Issue by olibclarke** [HappenApps/Quiver#1432]
Hi,

It seems like the full text search does not search the contents of titles.

I searched for a note which I knew had a certain word in the title, and it did not come up; other notes that had the same word in the body did come up.

I manually found the note and confirmed it's existence. When I added the same keyword to the body of the note, it then appeared in search results.

Bugs in search really need to be fixed, there are several and it really impacts usability.

@ylian is it possible to get an updated timeline on when we might expect a new version of the app for either Mac or iOS? I would happily pay again for both, I have got an enormous amount of use out of the Mac app.

Cheers
Oli

**Comment by cupdike** [comment]
Strong second on this. Titles word and word fragment matches should be given a high weighting relative to document body since titles typically summarize content--and are likely to be what we type in search for that content.

**Comment by cupdike** [comment]
Also from a UX perspective, suggest some kind of new user tip when first searching that points out the #tag special search.  I suffered with frustrating search for a long time before I realized tags had a special flag.  It's easy for new users to miss that--and it's really important for good search IMO.  Nothing is worse than not finding content you know is in there but not retrieved by search... and then manually trying to dig it out.  One aspect that leads to missing this feature--the time it takes to have enough notes that you need tag search is so long from when you get started, it's easy to have forgotten about the feature by the time you really start needing it.  

Another way to promote this is would be to detect users struggling with search (never using tag search, scrolling way down the list, running multiple similar searches, etc) and then pop up a tag search tip.

**Comment by junhocho** [comment]
Why is this still not working?

**Comment by cupdike** [comment]
This app has become a backwater of sorts. Last update was Sept 2019.  It still runs, and I use it for some things with its faults, but no longer recommend it to anyone.

**Comment by olibclarke** [comment]
Agreed - I switched over to Bear, which was a pretty seamless transition. I miss some things about quiver, particularly the way syntax coloring is handled, but it gets the job done. And search and sync actually work.

**Comment by yorch** [comment]
I've been looking at [Joplin](https://github.com/laurent22/joplin), which is open source and from my limited testing works great and looks awesome. Unfortunately there isn't an easy way to export / import all of my notebooks from Quiver to it, as I would like to maintain creation / update dates and all metadata. Might need to spend some time contributing a Quiver importer to Joplin.

**Comment by cupdike** [comment]
A lot of apps accept markdown.  Here'a tool that bulk exports to markdown (it looks like it was recently updated too):
https://github.com/mbelletti/quiver-tools

I haven't tried it yet.  If anyone tries this, please report back how it goes (especially interested in Joplin and Bear).  TIA

**Comment by yorch** [comment]
@cupdike yeah, even Quiver can export complete notebooks in different formats like HTML and Markdown (one note per md file):

![image](https://user-images.githubusercontent.com/60499/111657981-d97c1800-8803-11eb-9881-76a2adf47bf7.png)

But what I need is a way to export my notes including metadata (like tags, creation and update datetimes, etc), as I would like to preserve history and all that.
